src/components/CoordinatorMenu.js

```diff
@@ -28,37 +28,13 @@ const CoordinatorMenu = observer(({ assembly }) => (
             </Toggle>
 
             <Question>
-              <label htmlFor="name">
-                {assembly.translate("menu.name")}
-              </label>
-
-              <Field
-                name="name"
-                value={assembly.coordinator_account.information.name || ""}
-                onChange={(e) => {
-                  assembly.coordinator_account.information.name = e.target.value
-                  assembly.coordinator_account.update(assembly.uuid)
-                }}
-              />
+              {field(assembly, "coordinator_menu.name").label}
+              {field(assembly, "coordinator_menu.name").field}
             </Question>
 
             <Question>
-              <label htmlFor="email">
-                {assembly.translate("menu.email")}
-              </label>
-
-              <Field
-                name="email"
-                type="email"
-                value={assembly.coordinator_account.information.email || ""}
-                onChange={(e) => {
-                  // TODO possible race condition.
-                  // Fold all logic into the `update` function,
-                  // with a semaphore to prevent losing data.
-                  assembly.coordinator_account.information.email = e.target.value
-                  assembly.coordinator_account.update(assembly.uuid)
-                }}
-              />
```

Comment:
Here's our big win - 12 lines of code replaced by 2, repeated many places throughout the software.